### PR TITLE
initial mixer commit

### PR DESCRIPTION
**Pull request by akolesnikoff**

### CODE REVIEW

File: vit_jax/configs/mixer_base16_cifar10.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC.
```
Comment (andsteing) [inline vit_jax/configs/mixer_base16_cifar10.py:1]: Two copyright notes.

File: vit_jax/configs/mixer_base16_cifar10.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+def get_config():
+  """Returns config for training Mixer-B/16 on cifar10."""
+  config = common.get_config()
+  config.model_type = 'Mixer'
+  config.model = models.get_mixer_b16_config()
+  config.dataset = 'cifar10'
+  config.total_steps = 10_000
+  config.pp = ml_collections.ConfigDict(
```
Comment (andsteing) [inline vit_jax/configs/mixer_base16_cifar10.py:41]: Consider using `common.with_dataset()`

File: vit_jax/train_test.py [hunk 3]
```diff
@@ -45,7 +45,6 @@ def test_train_and_evaluate(self):
 
     test_utils.create_checkpoint(config.model, f'{workdir}/testing.npz')
     opt_pmap = train.train_and_evaluate(config, workdir)
-    opt_pmap = train.train_and_evaluate(config, workdir)
```
Comment (andsteing) [inline vit_jax/train_test.py:48]: thanks :)

File: vit_jax/configs/mixer_base16_cifar10.py (hunk 1 above)
Comment (akolesnikoff) [reply vit_jax/configs/mixer_base16_cifar10.py]: done

File: vit_jax/configs/mixer_base16_cifar10.py (hunk 2 above)
Comment (akolesnikoff) [reply vit_jax/configs/mixer_base16_cifar10.py:27]: I have custom resolution, so will keep explicit configuration.

**Pull request merged**